app.js

```diff
@@ -1,10 +1,14 @@
 const express = require("express")
 
+const endpoints = require('./endpoints.json')
 const { getAllTopics } = require("./controllers/topics.controllers")
 
 const app = express()
 
 app.get("/api/topics", getAllTopics)
+app.get("/api", (req,res,next)=>{
+    res.status(200).send({endpoints})
+})
```

Comment:
To keep the top level file for you application ( `app.js` ) as clean as possible, you could extract this logic to a controller function